roles/template_service_broker/tasks/install.yml

```diff
@@ -28,10 +28,24 @@
     - "{{ __tsb_template_file }}"
     - "{{ __tsb_rbac_file }}"
     - "{{ __tsb_broker_file }}"
+    - "{{ __tsb_config_file }}"
+
+- yedit:
+    src: "{{ mktemp.stdout }}/{{ __tsb_config_file }}"
+    key: templateNamespaces
+    value: "{{ openshift_template_service_broker_namespaces }}"
+    value_type: list
+
+- slurp:
```

Comment:
why do we need to slurp the file? Won't `yedit` update the src file unless a backup is specified?

Reply:
yedit updates the src (the file in /tmp on the remote machine); we then need the contents of the file, which is passed in on the command line of oc process.

Reply:
Right, if the file in the temp dir is modified we don't need to slurp it because we could do a lookup of that modified file. I feel like this is just adding an extra step and then adding more steps later on to decode and format the output later.

Reply:
Ack, I always forget that it'll be a remote file...